item-spec/item-spec.md

```diff
@@ -183,14 +187,6 @@ Like the Link `rel` field, the `role` field can be given any value, however here
 | overview  | An asset that represents a possibly larger view than the thumbnail of the Item , for example, a true color composite of multi-band data. |
 | metadata  | A metadata sidecar file describing the data in this item, for example the Landsat-8 MTL file. |
 
-#### Asset types
-
-The following types are common for assets and are used as the key for the Asset Object:
-
-| Type      | Description |
-| --------- | ----------- |
-| thumbnail | STRONGLY RECOMMENDED. A downsampled image of the core asset, for direct display online in a web page or interactive search application. Even assets that are less easily translated in to a visual image should provide some visual representation, that users can browse. For example a SAR asset can render an elevation mask or hillshade for display. If at all possible it should be included for a better user experience in searching data. |
```

Comment:
I'm cool with moving this out, but I think we need to add that it's 'strongly recommended' in the 'roles' section. After that's done I'm up to approve this.

Reply:
Added it.